### DISCUSSION TITLE
Filament training and picking within cryoSPARC

### DISCUSSION THREAD

**Discussion by nyyallstar07** [tbepler/topaz#155] [cryoSPARC]
Hello, can anyone suggest how to pick filaments with Topaz in cryoSPARC? I trained a model using good particles, but in Topaz Extract I don't see any options to select a "filament mode" and define parameters such as inter-particle distance. Maybe I need to perform the training and picking outside of cryoSPARC to access filament features?

Thanks,
Peter

**Comment by marknakasone85** [comment]
I thought the filament tracer in cryosparc v4+ was ok. It should also work after TOPAZ denoise. https://guide.cryosparc.com/processing-data/all-job-types-in-cryosparc/particle-picking/job-filament-tracer-beta
